src/main/java/bio/terra/service/dataset/Dataset.java

```diff
@@ -139,15 +139,6 @@ public Dataset defaultProfileId(UUID defaultProfileId) {
         return this;
     }
 
-    public List<UUID> getAdditionalProfileIds() {
```

Comment:
Nit: Rebase on dev so this changeset is excluded? It would make reviewing the dataset doa refactor a little bit easier (instead of having to click on the individual commits).